tests/infra/base_smoketest.py

```diff
@@ -150,6 +150,12 @@ def post_search_es(self, replica, es_query):
         """ post-search using es, returns post-search response """
         return run_for_json(f'{self.venv_bin}hca dss post-search  --es-query {es_query} --replica {replica.name}')
 
+    def put_subscription(self, replica, subscription_type, query, url):
```

Comment:
Consider inlining `subscription_put_jmespath` and `subscription_put_es` into this method.

Reply:
ah thanks for this, reminded me i had to refactor the prod_smoketest 🙌